src/pages/haeriRoh/Login/LoginHr.scss

```
@@ -0,0 +1,135 @@
+$fa-color: #fafafa;
+$wh-color: white;
+$focus-line: 1px solid gray;
+$border-style: 1px solid #dbdbdb;
+
+@mixin flex-center {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
+
+/* css code */
+* {
+  box-sizing: border-box;
+}
+
+.main {
+  width: 100%;
+  height: 100vh;
+  @include flex-center();
+  background-color: $fa-color;
+
+  .container {
+    width: 340px;
+    @include flex-center();
+  }
+
+  .lineBox {
+    width: 320px;
+    height: 370px;
+    border: $border-style;
+    background-color: $wh-color;
+  }
+
+  .contentsBox {
+    width: 100%;
+    height: 100%;
+
+    .logo {
+      width: 100%;
+      height: 130px;
+      @include flex-center();
+
+      .logoName {
+        font-family: 'Lobster', cursive;
+        font-size: 37px;
+      }
+    }
+
+    .loginBox {
+      width: 100%;
+      height: 150px;
+      display: flex;
+      flex-direction: column;
+      justify-content: space-around;
+
+      .inputs {
+        width: 100%;
+        height: 80px;
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        align-items: center;
+
+        .input1 {
+          width: 100%;
+          display: flex;
+          justify-content: center;
+
+          .id {
+            width: 250px;
+            height: 37px;
+            border-radius: 3px;
+            border: $border-style;
+            background-color: $fa-color;
+          }
+
+          .id:focus {
+            outline: $focus-line;
+          }
+
+          .pw {
+            width: 250px;
+            height: 37px;
+            border-radius: 3px;
+            border: $border-style;
+            background-color: $fa-color;
+          }
+
+          .pw:focus {
+            outline: $focus-line;
+          }
+        }
```

```suggestion
          .pw {
            width: 250px;
            height: 37px;
            border-radius: 3px;
            border: $border-style;
            background-color: $fa-color;
   
          &:focus {
            outline: $focus-line;
          }
        }
```
scss 기능 중 &을 사용하면 위와 같이 수정할 수 있습니다~